lib/metric_fu/metrics/reek/generator.rb

```diff
@@ -1,6 +1,6 @@
 module MetricFu
   class ReekGenerator < Generator
-    REEK_REGEX = /^(\S+) (.*) \((.*)\)$/
+    REEK_REGEX = /^\s*\[([^:]+)\]:(\S+)\s+(.*)\s+\((.*)\)$/
```

Comment:
For any regex that approaches this level of omg I can't read this, please use the expanded `/x` form.

e.g.


```ruby
 REEK_REGEX = /
  ^ # start of line
    \s* # any number of spaces
    \[ # opening bracket
      ([^:]+) # one or more non- colon/closing bracket
    \] # closing bracket
    :  # colon
    (\S+) # one or more non-space characters
    \s+ # one or more spaces
    (.*) # anything
     \s+ # one or more spaces
    \(  # opening paren
       (.*)  # anything
     \)  # closing paren
   $ #end of line
  /x
```

(with more useful comments, and perhaps line breakdowns, and aligning the comments.. I just did this quickly).

If you wish, a better solution would be to just use the `Reek` library directly, or maybe look how projects like [rubycritic, quality, and pronto use it](https://github.com/metricfu/metric_fu/wiki/Code-Tools).